src/components/typography/label-large.tsx

```diff
@@ -1,10 +1,13 @@
+import { TypographyProps } from 'boilerplate-react-native/src/types/typography';
```

Comment:
It feels a bit off to take a dependency on third party lib. The goal is to build our own component using what RN provides.